lib/puppet-lint/data.rb

```diff
@@ -116,16 +116,16 @@ def resource_indexes
                 real_idx = token_idx + idx + 1
                 if tokens[real_idx].type == :LBRACE
                   depth += 1
-                elsif {:SEMIC => true, :RBRACE => true}.include? tokens[real_idx].type
+                elsif [:SEMIC, :RBRACE].include? tokens[real_idx].type
```

Comment:
The reason I went with hashes here is because Array#include? is O(n) and Hash#include? is O(1). If we're going to clean this up, I'd like to see this become a Set instead of an Array so that we don't slow things down.